app/views/application/menus/_user_admin_menu.html.erb

```diff
@@ -1,8 +1,8 @@
-<li class="dropdown">
+<li class="dropdown user-menu">
```

Comment:
If you're using BEM this should be a modifier class of the dropdown pattern, so I would do something like `<li class="dropdown dropdown--user-menu">`.

Otherwise, if you consider the user menu as a standalone pettern (I mean, that can eventually lives outside the dropdown pattern) I would do something like:

```
<li class="dropdown">
  <a class="dropdown-toggle" href="#" data-toggle="dropdown">
    <div class="user-menu">
      <%= image_tag avatar_url(current_user), class: "user-avatar" %>
      <span class="user-name"><%= current_user.email %></span>
      <i class="material-icons user-down">keyboard_arrow_down</i>
    </div>
  </a>
  ...
</li>